packages/runtime/src/templates/getCacheHandler.ts

```diff
@@ -0,0 +1,43 @@
+import { join } from 'path'
+
+import { copyFile, writeFile } from 'fs-extra'
+import { outdent as javascript } from 'outdent'
+
+export const generateCacheHandler = async (publish: string): Promise<void> => {
+  const blobCache = getCacheHandler()
+
+  await writeFile(join(publish, 'netlify-incremental-cache.js'), blobCache)
+  // node_modules is relative to the root of the project or monorepo
+  await copyFile(
```

Comment:
This file copy works locally, but the `process.cwd` is the `/Users/.../packages/next-runtime` folder. Just getting this sorted as it should be relative to the project root or monorepo root.